CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # VERSION HISTORY
 
-- 1.4.15 (in progress)
+- 1.5 (in progress)
+  - Fix an assertion error in chrF (#121)
+  - Add missing `__repr__()` methods for BLEU and TER
+  - TER: Fix exception when `--short` is used (#131)
+  - Pin Mecab version to 1.0.3 for Python 3.5 support
+  - [API Change]: Default value for `floor` smoothing is now 0.1 instead of 0.
+  - [API Change]: `sacrebleu.sentence_bleu()` now uses the `exp` smoothing method
+    similar to the CLI's --sentence-level behavior.
+  - Add smoothing value to BLEU signature (#98)
```

Comment:
Not just "similar", but the same! Maybe worth noting this was done to bring make their behavior the same

Reply:
done